recipes/openpmd-api/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "openpmd-api" %}
+{% set version = "0.1.1a" %}
+{% set sha256 = "039a0167a334f07d4dbd4f774880152534ace2ec830ab15343133fa8acd3a7be" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: 0.1.1-alpha.tar.gz
+  url: https://github.com/openPMD/openPMD-api/archive/0.1.1-alpha.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # Python 2.7 stack is build with too ancient vc9 (VS2008)
+  skip: True  # [win and py27]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pybind11 >=2.2.1
+    - cmake >=3.10
+    - adios >=1.13.1  # [unix]
+    - hdf5  >=1.8.13
+  run:
+    - python
+    - adios >=1.13.1  # [unix]
+    - hdf5  >=1.8.13
+
+test:
+  requires:
+    - numpy
+  imports:
+    - openPMD
+  commands:
+    - python -c "import openPMD; print(openPMD.__version__);"
+    - python -c "import openPMD; print(openPMD.variants)"
+    - test -f ${PREFIX}/lib/cmake/openPMD/openPMDConfig.cmake             # [unix]
+    - if exist %PREFIX%\cmake\openPMDConfig.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://www.openPMD.org
+  license: LGPL 3
+  license_family: LGPL
+  license_file: COPYING.LESSER
+  summary: 'C++ & Python API for writing & reading (H5, BP, ...), serial & MPI parallel openPMD files.'
+
+  description: |
+    This library provides a common high-level API for openPMD writing and
+    reading. It provides a common interface to I/O libraries and file formats
+    such as HDF5 and ADIOS.
+    This conda-forge package only provides serial I/O, please build from source
+    for MPI parallel I/O on HPC systems. Window support is limited to HDF5 and
+    Python 3.
+  doc_url: https://openpmd-api.readthedocs.io
+  dev_url: https://github.com/openPMD/openPMD-api
+
+extra:
+  recipe-maintainers:
+    - ax3l
+    - C0nsultant
```

Comment:
@C0nsultant - are you OK with being a maintainer?

Reply:
yes he confirmed above :) https://github.com/conda-forge/staged-recipes/pull/6032#issuecomment-395050766